eslint-config-fusion/index.js

```diff
@@ -30,6 +30,16 @@ module.exports = {
     'react-hooks/rules-of-hooks': 'error',
     // https://github.com/facebook/react/issues/14920
     'react-hooks/exhaustive-deps': 'warn',
+
+    // Fix inconsistency between Flow (inherited rule from flowtype/recommended) and Prettier
```

Comment:
Inherited rule:

https://github.com/gajus/eslint-plugin-flowtype/blob/712d840e0e72396a12a3e44021227f555306b75c/src/configs/recommended.json#L25-L28

The first option is whether to require a space or not after the type annotation colon (`"always"` in this case).

The second being whether a line break counts as a space following the annotation colon.  This one is key, as it's where the rule conflicts with Prettier.  If this is set to `"false"`, then when Prettier reformats the type to be on a new line, we conflict with this rule.